### ISSUE TITLE
php 5.3.3: file_get_contents "https" wrapper missing

### ISSUE THREAD

**Issue by Krinkle** [travis-ci/travis-ci#2895]
It works fine on Travis workers with php 5.3 (PHP 5.3.29), but those for php 5.3.3 are failing. Maybe configuration got out of sync (or defaults changed).

https://travis-ci.org/Krinkle/php-cssjanus/jobs/38699747

```
PHP 5.3.

PHP Warning:  file_get_contents(): Unable to find the wrapper "https" - did you forget to enable it when you configured PHP? in /home/travis/build/Krinkle/php-cssjanus/test/suites/CSSJanusTest.php on line 60
```

See also:
- http://stackoverflow.com/q/18643572/319266
- http://stackoverflow.com/q/1975461/319266

**Comment by stof** [comment]
@BanzaiMan is there any chance to rebuild the 5.3.3 image with openssl enabled thanks to the phpbuild patch ?

**Comment by BanzaiMan** [comment]
Sorry, but I am not sure how to make use of the patch as is.

Also keep in mind that phpunit is no longer available for install via PEAR, so that we cannot install it, even if we are able to compile PHP 5.3.x (and 5.2.x) the source. (It might be possible to pry it from the binary archive, but I don't know how usable it would be.)

**Comment by fabiang** [comment]
Are those required patches included in the definition file for 5.3.3?

```
patch_file "xp_ssl.c.patch"
patch_file "zip_direct.c.patch"
```

**Comment by stof** [comment]
@BanzaiMan just take the PHPUnit phar rather than getting it through PEAR (like for other versions of PHP)
